sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -58,7 +57,7 @@ export class CertificateClient {
     deleteContacts(options?: DeleteContactsOptions): Promise<CertificateContact[] | undefined>;
     deleteIssuer(issuerName: string, options?: DeleteIssuerOptions): Promise<CertificateIssuer>;
     getCertificate(certificateName: string, options?: GetCertificateOptions): Promise<KeyVaultCertificateWithPolicy>;
-    getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<KVPollerLike<CertificateOperationState, KeyVaultCertificateWithPolicy>>;
+    getCertificateOperation(certificateName: string, options?: GetCertificateOperationOptions): Promise<PollerLike<CertificateOperationState, KeyVaultCertificateWithPolicy>>;
```

Comment:
@jeremymeng Does a similar change apply to storage as well? If we are anyway updating the dependency version for storage, I would recommend we use the standard `PollerLike` in storage as well.

Reply:
Same question for form-recognizer

Reply:
Thank you Ramya! I thought they were using the standard PollerLike

Reply:
Right. `PollerLike` is used in Storage and Form Recognizer.

Reply:
Thanks for the confirmation @jeremymeng